packaging/src/main/assembly/src.xml

```diff
@@ -100,12 +100,26 @@
         <include>storage-api/**/*</include>
```

Comment:
This is unrelated to the changes here but it is kind of weird that we are packing in the source packages of Hive a project/module (`storage-api`) that has it's own release cycle. If we are including the sources here then we are somewhat releasing the storage-api multiple times.